services/harmony/app/frontends/ogc-edr/get-data-for-point.ts

```diff
@@ -0,0 +1,152 @@
+import { keysToLowerCase } from '../../util/object';
```

Comment:
Currently these functions are in the `get-data-for-point` module as I don't know how much of it would be refactored out when we add lines, etc. I would push the refactoring out when we add other geometries.